app/src/main/java/com/duckduckgo/app/notification/AndroidNotificationScheduler.kt

```diff
@@ -52,30 +58,65 @@ class NotificationScheduler(
 
     private suspend fun scheduleInactiveUserNotifications() {
         workManager.cancelAllWorkByTag(UNUSED_APP_WORK_REQUEST_TAG)
-
-        scheduleUnusedAppNotifications(
-            PRIVACY_DELAY_DURATION_IN_DAYS to TimeUnit.DAYS,
-            CLEAR_DATA_DELAY_DURATION_IN_DAYS to TimeUnit.DAYS,
-        )
-    }
-
-    private suspend fun scheduleUnusedAppNotifications(
-        privacyDelayTime: Pair<Long, TimeUnit>,
-        clearDataDelayTime: Pair<Long, TimeUnit>,
-    ) {
-        if (privacyNotification.canShow()) {
-            scheduleNotification(
-                OneTimeWorkRequestBuilder<PrivacyNotificationWorker>(),
-                privacyDelayTime.first,
-                privacyDelayTime.second,
-                UNUSED_APP_WORK_REQUEST_TAG,
-            )
+        when {
+            variantManager.isCompetitiveCopyEnabled() -> {
```

Comment:
are we doing the same thing in `variantManager.isSetupCopyCopyEnabled()`? if so, maybe we can avoid duplicate this part?